diff -urNp gengetopt-2.19.1.orig/doc/README.example gengetopt-2.19.1/doc/README.example
--- gengetopt-2.19.1.orig/doc/README.example	1970-01-01 05:30:00.000000000 +0530
+++ gengetopt-2.19.1/doc/README.example	2007-06-10 16:57:03.000000000 +0530
@@ -0,0 +1,12 @@
+#!/bin/sh -v
+
+# commands to try the C++ example:
+
+gengetopt -isample1.ggo -Fcmdline1 --long-help -u c++ -o main1 main1.cc cmdline1.c
+./main1 --help
+
+# commands to try the C example: 
+
+gengetopt --input=sample2.ggo --func-name=my_cmdline_parser --file-name=cmdline2 --unamed-opts
+gcc -o main2 main2.c cmdline2.c
+./main2 -h
